c2cgeoportal/views/entry.py

```diff
@@ -1276,11 +1277,12 @@ def logout(self):
         return set_common_headers(
             self.request, "logout", NO_CACHE,
             response=Response("true", headers=headers),
+            add_cors=True
         )
 
     @view_config(route_name="loginchange", renderer="json")
     def loginchange(self):
-        set_common_headers(self.request, "loginchange", NO_CACHE)
+        set_common_headers(self.request, "loginchange", NO_CACHE, add_cors=True)
```

Comment:
This means a CSRF could now be used to change the user's password. Same thing for loginresetpassword.